### ISSUE TITLE
3 bugs for direct using

### ISSUE THREAD

**Issue by guangyuli-uoe** [Lightning-AI/litgpt#2098]
### Bug description

1. when pip install -e '.[all]', there is warning: WARNING: litgpt 0.5.9 does not provide the extra 'all'
2. when do cpt with the provided example codes, 
```
litgpt pretrain EleutherAI/pythia-160m \
  --tokenizer_dir EleutherAI/pythia-160m \
  --initial_checkpoint_dir EleutherAI/pythia-160m \
  --data TextFiles \
  --data.train_data_path "custom_texts/" \
  --train.max_tokens 10_000_000 \
  --out_dir out/custom-model
```
the loss is increasing...

3. when use qwen2.5-0.5b and qwen2.5-0.5b-instruct downloaded by litgpt download, it raises a cuda error: "cuda driver error: invalid argument"

### What operating system are you using?

Unknown

**Comment by guangyuli-uoe** [comment]
what i did for the env:

1. download the zip file for the latest version of litgpt
2. unzip
3. pip install -e '.[all]'
4. pip install litdata
5. pip install tensorboard

not quite sure if the version of python is conflict, but i tried the following py-verstion:
3.11.13
3.10.18
3.11.0
3.9.22
3.12.11

**Comment by guangyuli-uoe** [comment]
here are some details when i run this command:

`

litgpt pretrain Qwen/Qwen2.5-0.5B-Instruct \
  --tokenizer_dir /litgpt-main/checkpoints/Qwen/Qwen2.5-0.5B-Instruct \
  --initial_checkpoint_dir/litgpt-main/checkpoints/Qwen/Qwen2.5-0.5B-Instruct \
  --data TextFiles \
  --data.train_data_path "custom_texts/" \
  --train.max_tokens 10_000_000 \
  --out_dir out/custom-model
`

<img width="2078" height="1090" alt="Image" src="https://github.com/user-attachments/assets/e8cc5972-40c6-4d73-9fea-f9f9f548da1d" />

**Comment by guangyuli-uoe** [comment]
here are loss details when i run cpt command in the readme file:

`

litgpt pretrain EleutherAI/pythia-160m \
  --tokenizer_dir EleutherAI/pythia-160m \
  --initial_checkpoint_dir EleutherAI/pythia-160m \
  --data TextFiles \
  --data.train_data_path "custom_texts/" \
  --train.max_tokens 10_000_000 \
  --out_dir out/custom-model
`

<img width="1632" height="526" alt="Image" src="https://github.com/user-attachments/assets/9670ed0b-59e5-44ee-9d0d-f90ec84bbc7c" />

**Comment by guangyuli-uoe** [comment]
@Borda 

hi, 
i tried torch in different version(2.7.1 and 2.5.1), but the "cuda driver error: invalid argument" is still there,
and it seems that this only occurs when i use qwen2.5-0.5b and qwen2.5-0.5b-instruct,

looking forward to your kind reply !

best wishes,

**Comment by guangyuli-uoe** [comment]
and im pretty sure i could train qwen2.5-0.5b and qwen2.5-0.5b-instruct before july，

**Comment by NamburiSrinath** [comment]
Hi @guangyuli-uoe 

I am facing similar issue and suspect the issue is with tensorboard. Do you have any solution?
